laboratorio.html

```diff
@@ -0,0 +1,358 @@
+                            <!--/Header-->
+
+                            <!--Computer schedule Cont-->
+                            <div class="horario-container">
+                                <h6 class="m-0 font-weight-bold text-dark pb-3">Horario de la computadora</h6>
+                                <!--schedule Details-->
+                                <div class="horario-Container">
+
+                                </div>
+                                <!--schedule Details-->
+                            </div>
+                            <!--/Computer schedule Cont-->
+
+                            <!--Software Cont-->
+
+                            <div class="Software-container">
+                                <h6 class="m-0 font-weight-bold text-dark pb-3">Software Instalado</h6>
+                                <!--Software Details-->
+                                <div class="Software-Container-sm mb-4">
+                                    <!--Software Row-->
+                                    <div class="row">
+                                        <div class="col"><i class="fa-solid fa-file-word fa-xl"
+                                                style="color: #0658e5;"></i></div>
+                                        <div class="col"><i class="fa-solid fa-file-excel fa-xl"
+                                                style="color: #007542;"></i></div>
+                                        <div class="col"><i class="fa-solid fa-file-powerpoint fa-xl"
+                                                style="color: #f26202;"></i></div>
+
+                                        <div class="col"><i class="fa-solid fa-compass-drafting fa-xl"
+                                                style="color: #0000ff;"></i></div>
+                                        <div class="col"><i class="fa-solid fa-file-image fa-xl"
+                                                style="color: #00ffff;"></i></div>
+                                        <div class="col"><i class="fa-brands fa-artstation fa-xl"
+                                                style="color: #ff0000;"></i></div>
+                                    </div>
+                                    <!--/Software Row-->
+                                </div>
+                                <!--/Software Details-->
+
+                                <!--ButtonsCont-->
+                                <div class="buttons-container">
+                                    <div>
+                                        <button type="button" class="btn btn-primary mb-3">Solicitar</button>
+                                    </div>
+                                    <button type="button" class="btn btn-primary">Modificar</button>
+                                    <button type="button" class="btn btn-secondary">Clonar</button>
+                                    <button type="button" class="btn-danger">Eliminar</button>
```

Comment:
Le falta la clase de `btn`.

```suggestion
                                    <button type="button" class="btn btn-danger">Eliminar</button>
```